src/themes/default.ts

```diff
@@ -171,8 +171,41 @@ export const defaultTheme = {
   },
   typography: {
     fontFamily: 'Roboto, "Open Sans", arial, sans-serif',
-    fontSize: '16px',
-    color: defaultColors.black,
+    fontSize: '14px',
```

Comment:
Let's add the option for a `secondaryFontFamily` here. It won't be used for our default theme, but we've needed that before.